app/ide-desktop/lib/content/src/index.html

```diff
@@ -27,12 +27,14 @@
             user-scalable = no"
     />
     <title>Enso</title>
+    <script src="https://cdn.tailwindcss.com"></script>
```

Comment:
we can't do it. The app should work without internet connection, so you can't download things at runtime.

Reply:
Fixed. TailwindCSS is now generated and output to a file.